src/data/nat/prime.lean

```diff
@@ -613,6 +613,20 @@ begin
   exact le_antisymm h (not_le.1 ((not_congr (pow_dvd_pow_iff_le_right (prime.one_lt pp))).1 h₁)),
 end
 
+lemma ne_one_iff : ∀ {n}, n ≠ 1 ↔ ∃ p : ℕ, p.prime ∧ p ∣ n
```

Comment:
This is not the first think I would expect from `nat.ne_one_iff`. Perhaps `ne_one_iff_exists_prime_dvd` would be better?

Reply:
sure, I wasn't sure what else I would expect from it either, though.